npf/types/notebook/template.ipynb

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
Can this template be changed by the user easily? The path to this should be an overwriteable default.

Reply:
Ok, working on it.